### PR TITLE
[Improvement]: Add multiselect filtering support on grid for classification store

### PR DESCRIPTION
**Pull request by msoroka**
## Changes in this pull request  
Add support for multiselect filter on grid when field is provided by classification store.

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/Multiselect.php [hunk 1]
```diff
@@ -458,6 +458,14 @@ public function getFilterConditionExt($value, $operator, $params = [])
                 $key = $params['brickPrefix'].$key;
             }
 
+            if (str_contains($name, 'cskey') && is_array($value) && !empty($value)) {
+                $values = array_map(function ($val) {
+                    return "'%" . addslashes($val) . "%'";
```
Comment (jdreesen) [inline models/DataObject/ClassDefinition/Data/Multiselect.php]: I don't think this is appropriate escaping of values that are used in a DB query.

Comment (robertSt7) [reply models/DataObject/ClassDefinition/Data/Multiselect.php]: @msoroka How is it going about this? Please have a look at https://github.com/pimcore/pimcore/blob/11.x/models/Asset/Dao.php#L204 And could you also please merge 10.5 in your branch to fix the tests? Thanks

Comment (msoroka) [reply models/DataObject/ClassDefinition/Data/Multiselect.php]: @jdreesen @robertSt7 Thanks for suggestions

**Comment by msoroka** [comment]
@robertSt7 @jdreesen I made some changes

**Comment by msoroka** [comment]
@robertSt7 all checks have passed

**Comment by robertSt7** [comment]
@msoroka Thanks a lot for the PR

**Pull request merged**